.circleci/config.yml

```diff
@@ -43,7 +43,9 @@ jobs:
     working_directory: /home/circleci/go/src/github.com/influxdata/influx-cli
     steps:
       - checkout
-      - run: make influx
+      - run: make crossbuild
+    store_artifacts:
```

Comment:
I believe this needs a `-` in front of it to make it one of the steps of the job